controller/search_blackbox_test.go

```diff
@@ -1204,8 +1204,8 @@ func (s *searchControllerTestSuite) TestIncludedParents() {
 				// check order of execution
 				if len(result.Included) > 1 {
 					var expectedOrder []interface{}
-					for i := 0; i < 2; i++ {
-						expectedOrder = append(expectedOrder, fxt.WorkItems[i].ID) // expectedOrder = [A, B]
+					for i := 1; i >= 0; i-- {
+						expectedOrder = append(expectedOrder, fxt.WorkItems[i].ID) // expectedOrder = [B, A]
```

Comment:
@DhritiShikhar it is hard to believe that this change is all that it takes... ;)

Reply:
@kwk this is still in progress. This change does not sort the workitems properly. I am still working on it. But yes, the workitem children right now are sorted based on title/ID which I am trying to replace with order of execution